client/landing/gutenboarding/analytics.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { useEffect } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { useOnUnmount } from 'landing/gutenboarding/hooks/use-on-unmount';
```

Comment:
For consistency should these imports use a relative path instead? (I think there was an idea that we might eventually move Gutenboarding from `landing/gutenboarding` so relative paths might make that easier in the future)

Reply:
good call 👍